app/src/main/java/org/jellyfin/androidtv/util/TimeUtils.java

```diff
@@ -148,10 +160,18 @@ public static String getFriendlyDate(Date date, boolean relative) {
         cal.setTime(date);
         Calendar now = Calendar.getInstance();
         if (cal.get(Calendar.YEAR) == now.get(Calendar.YEAR)) {
-            if (cal.get(Calendar.DAY_OF_YEAR) == now.get(Calendar.DAY_OF_YEAR)) return TvApp.getApplication().getString(R.string.lbl_today);
-            if (cal.get(Calendar.DAY_OF_YEAR) == now.get(Calendar.DAY_OF_YEAR)+1) return TvApp.getApplication().getString(R.string.lbl_tomorrow);
-            if (cal.get(Calendar.DAY_OF_YEAR) < now.get(Calendar.DAY_OF_YEAR)+7 && cal.get(Calendar.DAY_OF_YEAR) > now.get(Calendar.DAY_OF_YEAR)) return cal.getDisplayName(Calendar.DAY_OF_WEEK, Calendar.LONG, Locale.getDefault());
-            if (relative) return String.format(TvApp.getApplication().getString(R.string.lbl_in_x_days), cal.get(Calendar.DAY_OF_YEAR) - now.get(Calendar.DAY_OF_YEAR));
+            if (cal.get(Calendar.DAY_OF_YEAR) == now.get(Calendar.DAY_OF_YEAR)) {
+                return TvApp.getApplication().getString(R.string.lbl_today);
+            }
+            if (cal.get(Calendar.DAY_OF_YEAR) == now.get(Calendar.DAY_OF_YEAR)+1) {
+                return TvApp.getApplication().getString(R.string.lbl_tomorrow);
+            }
```

Comment:
There are some spaces missing in these blocks for syntax.

Reply:
Fixed.